assisted-events-scrape/storage/cluster_events_storage.py

```diff
@@ -0,0 +1,146 @@
+import elasticsearch
```

Comment:
We've moved all related code to "Process and store events in elastic" to this module.
It's far from ideal has this hasn't a single responsibility, and we'll definitely tackle this in the future. For now we can consider this a "Laundry drawer" for "elastic-related-stuff".